community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/input/csv/CsvInput.java

```diff
@@ -107,11 +108,12 @@ private void checkUniqueCharacter( Map<Character,String> characters, char charac
             @Override
             public InputIterator<InputNode> iterator()
             {
-                return new InputGroupsDeserializer<>( nodeDataFactory.iterator(),
-                        nodeHeaderFactory, config, idType, maxProcessors, (dataStream, dataHeader, decorator) ->
+                DeserializerFactory<InputNode> factory = (dataStream, dataHeader, decorator, validator) ->
```

Comment:
Would be nice if we could agree on one ordering of `dataStream` and `dataHeader` and not have different ordering in different places.
(`InputEntityDeserializer` and `DeserializerFactory`)